packages/lambda-shared/src/aws/tile.metadata.table.ts

```diff
@@ -99,8 +99,17 @@ export class TileMetadataTable {
         }
 
         if (imageList.size > 0) {
-            const items = await this.fetchImagery(Array.from(imageList.values()));
+            const keys = Array.from(imageList.values());
+            const items = await this.fetchImagery(keys);
             for (const item of items) output.set(item.id, item);
+
+            if (items.length < keys.length) {
+                let ids = '';
```

Comment:
could simplify to something like 

`const ids = keys.filter(f => !output.has(f)).join(', ')`